slices/python3.11-venv.yaml

```
@@ -9,3 +12,9 @@ slices:
       - python3.11_standard
     contents:
       /usr/lib/python3.11/ensurepip/*.py:
+
+  copyright:
+    essential:
+      - python3.11_copyright
+    contents:
+      /usr/share/doc/python3.11-venv/: { symlink: /usr/share/doc/python3.11/ }
```

Same as `python3.10-venv` above.

```suggestion
      /usr/share/doc/python3.11-venv/:
```